app/views/works/new_import.html.erb

```diff
@@ -130,6 +122,49 @@
     </dl>
   </fieldset>
 
+  <fieldset>
+    <legend><%= t("works.permissions.privacy") %></legend>
+    <h3 class="landmark heading"><%= t("works.permissions.privacy") %></h3>
+    <dl>
+      <dt class="restricted">
+        <%= check_box_tag "restricted" %>
+      </dt>
+      <dd class="restricted">
+        <%= label_tag "restricted", t("works.permissions.visibility.import_works_restricted") %>
+        <%= link_to_help "registered-users" %>
+      </dd>
+
+      <dt class="moderated comments">
+        <%= check_box_tag "moderated_commenting_enabled" %>
+      </dt>
+      <dd class="moderated comments">
+        <%= label_tag :moderated_commenting_enabled, t("comments.commentable.permissions.moderated_commenting.enable") %>
+        <%= link_to_help "comments-moderated" %>
+      </dd>
+
+      <dt class="permissions comments">
+        <%= t("comments.commentable.permissions.options.label") %>
```

Comment:
Oh, you know what, the label here should probably say "Who can comment on these works" rather than "this work." Sorry for not realizing that in the issue!